src/components/ListItem/ListItem.tsx

```diff
@@ -46,14 +69,21 @@ export const ListItem: React.FC<IListItemProps> = ({
     );
 
     return (
-        <li className={classnames(cssListItem)}
+        <li
+            className={classnames(cssListItem)}
+            tabIndex={tabIndex}
             {...props}
         >
             {
-            iconStart &&
+                iconStart &&
```

Comment:
Yes, we can do that. It could be confusing for the user to have both the `iconStart` and the `image` disappear if both are specified but I can make a note in the documentation of the props.